src/triggers.js

```diff
@@ -161,6 +161,28 @@ export function _unregisterAll() {
   Object.keys(_triggerStore).forEach(appId => delete _triggerStore[appId]);
 }
 
+export function toJSONwithObjects(object, className) {
```

Comment:
I am not sure, but have you tried with `res.object._toFullJSON()` instead of using this function?

Reply:
Yes, `toFullJSON` does not work either

Reply:
Maybe this function could be a great addition to the SDK instead of being here? @dplewis thoughts?